fusequery/query/src/datasources/remote/store_client_provider.rs

```diff
@@ -9,8 +9,8 @@ use common_exception::Result;
 use common_flights::StoreClient;
 
 #[async_trait::async_trait]
-pub trait IStoreClientProvider {
+pub trait TryGetStoreClient {
```

Comment:
What if the trait is extended with more functions in the future?

Reply:
At that time, we should add another trait or refactor this trait, it depends, IMHO.

Reply:
Fine with me.